manifests/k/KiCad/KiCad/Lite/5.1.12_1/KiCad.KiCad.Lite.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: KiCad.KiCad.Lite
+PackageVersion: 5.1.12_1
+PackageLocale: en-US
+Publisher: KiCad
+PublisherSupportUrl: https://www.kicad.org/help/
+Author: KiCad
+PackageName: KiCad
+PackageUrl: https://www.kicad.org/
+License: GPL-3.0-or-later
+LicenseUrl: https://gitlab.com/kicad/code/kicad/-/blob/master/LICENSE
+Copyright: Copyright (c) 2014-2021 KiCad Developers
+ShortDescription: KiCad is a free software suite for electronic design automation. It facilitates the design of schematics for electronic circuits and their conversion to PCB designs.
+Moniker: kicad

```

Comment:
```suggestion
Moniker: kicad-lite
```

Reply:
I have copied this from the existing Lite package, but I agree, a different value should be assigned here.